### PR TITLE
Adding Support for CN0579

### PR DESCRIPTION
**Pull request by nsacut**
# Description

Created and tested Python Class, Example, and Test Scripts for CN0579.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Hardware used was EVAL-CN0579-ARDZ and Cora Z7 connected to laptop. Testing and development of python codes were done on a laptop running Windows 10.

**Test Configuration**:
* Hardware: EVAL-CN0579-ARDZ and Cora Z7
* OS: Windows 10

# Documentation

If this is a new feature or example please mention or link any documentation. All new hardware interface classes require documentation.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have signed off all commits and they contain "Signed-off by: Nathaniel Acut <nathaniel.acut@analog.com>"
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by nsacut** [comment]
Requesting a review from @tfcollins

File: examples/cn0579/sin_params.py
Comment (tfcollins) [inline examples/cn0579/sin_params.py:1]: This file already exists here: https://github.com/analogdevicesinc/pyadi-iio/blob/master/examples/ad4630/sin_params.py

File: adi/cn0579.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    def CC_CH0(self, value):
+        self._set_iio_attr_int("voltage0", "raw", True, value, self._gpio)
+
+    @property
+    def CC_CH1(self):
+        return self._get_iio_attr("voltage1", "raw", True, self._gpio)
+
+    @CC_CH1.setter
+    def CC_CH1(self, value):
+        self._set_iio_attr_int("voltage1", "raw", True, value, self._gpio)
```
Comment (tfcollins) [inline adi/cn0579.py:98]: Please add docstring

File: adi/cn0579.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+    @property
+    def CC_CH2(self):
+        return self._get_iio_attr("voltage2", "raw", True, self._gpio)
+
+    @CC_CH2.setter
+    def CC_CH2(self, value):
+        self._set_iio_attr_int("voltage2", "raw", True, value, self._gpio)
```
Comment (tfcollins) [inline adi/cn0579.py:106]: Please add docstring

File: adi/cn0579.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+    @property
+    def CC_CH3(self):
+        return self._get_iio_attr("voltage3", "raw", True, self._gpio)
+
+    @CC_CH3.setter
+    def CC_CH3(self, value):
+        self._set_iio_attr_int("voltage3", "raw", True, value, self._gpio)
```
Comment (tfcollins) [inline adi/cn0579.py:114]: Please add docstring

**Review by tfcollins — CHANGES_REQUESTED** [review]
Can you remove the comments for the lines that have the scaling if not used

**Comment by nsacut** [comment]
> Can you remove the comments for the lines that have the scaling if not used

Yes, removed the commented lines for the unused scaling.

**Comment by tfcollins** [comment]
Just fix the lint issue and we are good to merge

File: examples/cn0579/cn0579_example.py [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+ad4630_dir = os.path.join(parentdir, "ad4630")
+sys.path.insert(0, ad4630_dir)
+
+import sin_params
+
+# from save_for_pscope import save_for_pscope
+
+# Optionally passs URI as command line argument,
```
Comment (tfcollins) [inline examples/cn0579/cn0579_example.py:24]: Lint is failing on this spelling mistake

**Comment by nsacut** [comment]
> Lint issue not fixed yet

The commit https://github.com/analogdevicesinc/pyadi-iio/pull/450/commits/7f732822df135e96e7ef2c0d44589f47c92d7c64 should fix the remaining lint issues.

**Comment by nsacut** [comment]
Hi @tfcollins, all checks are passing now. :)

**Review by tfcollins — APPROVED** [review]

**Pull request merged**